journal_brief/config.py

```diff
@@ -292,6 +317,28 @@ def validate_email(self):
                 yield SemanticError('expected string', 'password',
                                     {'smtp': {'password': smtp['password']}})
 
+            if 'headers' in smtp:
+                    if not isinstance(smtp['headers'], dict):
+                        yield SemanticError('expected dict', 'headers',
+                                            {'smtp': {'headers': smtp['headers']}})
+
+                    if 'From' in smtp['headers']:
```

Comment:
What if it's `'from'`?

Reply:
Excellent point; I had forgotten that SMTP headers are not case sensitive. Fix on the way.